radar/2024-09-04/couch2pg.md

```diff
@@ -0,0 +1,12 @@
+---
+title:      "couch2pg"
+ring:       stop
+quadrant:   methods-and-patterns
+tags:       [data-use]
+---
+
+[couch2pg](https://github.com/medic/couch2pg) streams data from CouchDB and forwards it to PostgreSQL.
+
+An updated version of couch2pg is [embedded in CHT Sync repository](https://github.com/medic/cht-sync/tree/main/couch2pg) and no longer uses the [deprecated repository](https://github.com/medic/couch2pg). 
+
+For data synchronization, refer to [CHT Sync documentation](https://docs.communityhealthtoolkit.org/core/overview/cht-sync/).
```

Comment:
The naming here is really confusing. I think we should drop the term "couch2pg" altogether - it's fine if it lives on internally, but externally there is no such thing.

```suggestion
couch2pg is deprecated. For data synchronization, refer to [CHT Sync documentation](https://docs.communityhealthtoolkit.org/core/overview/cht-sync/).
```